sql/create.sql

```diff
@@ -1,72 +1,73 @@
 CREATE SCHEMA dbo; --POSTGRES ALREADY USES 'public' SCHEMA BY DEFAULT
 
-create table dbo.Programme(
-	id   		INT generated always as identity primary key,
-	acronym 	VARCHAR(10) unique,				
-	name		VARCHAR(100) UNIQUE,			-- It may be null in this phase
-	termSize	INT					
+CREATE TABLE dbo.Programme(
+	id   			INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	acronym 		VARCHAR(10) UNIQUE,				
+	name			VARCHAR(100) UNIQUE,			-- It may be NULL in this phase
+	termSize		INT					
 );
 
-create table dbo.Calendar (
-	id   int generated always as identity primary key
+CREATE TABLE dbo.Calendar (
+	id   			INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY
 );
 
-create table dbo.Course (
-	id   	 INT generated always as identity primary key,
-	acronym  varchar(10) unique,	
-	name     varchar(100) unique				-- It may be null in this phase
+CREATE TABLE dbo.Course (
+	id   	 		INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	acronym 		VARCHAR(10) UNIQUE,	
+	name     		VARCHAR(100) UNIQUE				-- It may be NULL in this phase
 );
 
 
-create table dbo.ProgrammeOffer(
-	id   	 		INT generated always as identity,
-	programmeId 		INT REFERENCES dbo.Programme(id),
+CREATE TABLE dbo.ProgrammeOffer(
+	id   	 		INT GENERATED ALWAYS AS IDENTITY,
+	programmeId 	INT REFERENCES dbo.Programme(id),
 	courseId 		INT REFERENCES dbo.Course(id),
 	termNumber		INT, 
 	optional		BOOLEAN,
 	PRIMARY KEY(programmeId, courseId, termNumber)
 );
 
-create table dbo.CalendarTerm (
-	id         varchar(10) primary key, -- e.g. "1920v"
-	start_date timestamp,
-	end_date   timestamp check(end_date > start_date),
-	unique(start_date, end_date)
+CREATE TABLE dbo.CalendarTerm (
+	id         		VARCHAR(10) PRIMARY KEY, -- e.g. "1920v"
+	start_date 		TIMESTAMP,
+	end_date   		TIMESTAMP check(end_date > start_date),
+	UNIQUE(start_date, end_date)
 );
 
-create table dbo.Class (
-	courseId INT references dbo.Course(id),
-	term     varchar(10) references dbo.CalendarTerm(id),
-	calendar int references dbo.Calendar(id),
-	primary key(courseId, term)
+CREATE TABLE dbo.Class (
+	id 				INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	courseId 		INT REFERENCES dbo.Course(id),
+	term     		VARCHAR(10) REFERENCES dbo.CalendarTerm(id),
```

Comment:
We forgot to change this to `calendarTerm`

Reply:
You mean just the column name?

Reply:
Yes